src/styles/sidebar/styled-text.scss

```diff
@@ -107,7 +108,7 @@
   }
 
   blockquote {
-    @include var.font-normal;
+    @include utils.font--medium;
 
     border-left: 3px solid var.$grey-3;
     color: var.$color-text-light;
```

Comment:
Can `font-*` css below can be removed? Are you intended to do this in 2nd passes? I'm not sure, I'll just call out what I can catch atm. Feel free to ignore if you have plans for this.